docs/api/combineManagers.md

```diff
@@ -0,0 +1,26 @@
+# `combineManagers(...managers)`
```

Comment:
I know the term but it's not cool - SPMC (single producer multiple consumers) ;-) from the world of lock free algorithms